src/plugins/fundingRequest/plugin.ts

```diff
@@ -138,7 +138,7 @@ export class FundingRequest
       method: 'propose',
       args: [
         options.beneficiary,
-        options.amount.toNumber(),
+        options.amount.toString(),
```

Comment:
this plugin is not supported in alchemy yet. so why update that ?
@jellegerbrandy is that needed for Common ?

Reply:
the change is needed in any case.  without it we would have a bug by casting a BN to a number, assuming the BN can easily overflow a `number`.

Reply:
you are asking me if the funding request plugin is needed for common?

yes, it is.

Reply:
Yeah, this is good 👍